### PR TITLE
Update IS_PROD check

### PR DESCRIPTION
**Pull request by haileyok**
`IS_PROD` should only be false if it's a local dev service of the bsky staging service.

### CODE REVIEW

File: src/lib/constants.ts [hunk 1]
```diff
@@ -36,92 +35,12 @@ export const MAX_GRAPHEME_LENGTH = 300
...
-  // this definition will not work once federation is enabled!
-  // -prf
-  return (
-    url.startsWith('https://bsky.social') ||
-    url.startsWith('https://api.bsky.app') ||
-    /bsky\.network\/?$/.test(url)
-  )
+export function IS_PROD_SERVICE(url?: string) {
+  return url && url !== STAGING_SERVICE && url !== LOCAL_DEV_SERVICE
 }
```
Comment (haileyok) [inline src/lib/constants.ts:40]: The only cases where we shouldn't be in prod are staging and local right?

File: src/lib/constants.ts [hunk 2]
```diff
@@ -135,13 +54,11 @@ export const STAGING_LINK_META_PROXY =
...
-    return STAGING_LINK_META_PROXY
-  } else if (IS_STAGING(serviceUrl)) {
-    return STAGING_LINK_META_PROXY
-  } else {
+  if (IS_PROD_SERVICE(serviceUrl)) {
     return PROD_LINK_META_PROXY
   }
+
+  return STAGING_LINK_META_PROXY
 }
```
Comment (haileyok) [inline src/lib/constants.ts:63]: Again, we should always use cardyb as long as we are not on staging regardless of service iiuc.

**Review by estrattonbailey — APPROVED** [review]
LGTM! Thanks for cleaning all that up. Just wanna get a triple check from @dholms on prod vs everything else logic.

**Review by dholms — APPROVED** [review]
yup looks right 👍

**Pull request merged**